Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -130,20 +136,20 @@ private class NpmPathProvider : INpmPathProvider {
             }
         }
 
-        private INpmController CreateNpmController() {
-            if (null == _npmController) {
```

Comment:
This method got turned from a method that sets a member variable, to a factory method. The actual member variable is set in the ctor now instead of inside of `CreateNpmController`.